integration_tests/utils.py

```diff
@@ -19,26 +19,26 @@ def wait_for_block(cli, height, timeout=60):
         except AssertionError as e:
             print(f"get sync status failed: {e}", file=sys.stderr)
         else:
-            if int(status["sync_info"]["latest_block_height"]) >= height:
+            if int(status["SyncInfo"]["latest_block_height"]) >= height:
```

Comment:
why changed to camel style?

Reply:
it's changed in cosmos-sdk in this PR: https://github.com/cosmos/cosmos-sdk/pull/7974/files